prime-router/src/main/kotlin/azure/RequestFunction.kt

```diff
@@ -50,14 +50,24 @@ abstract class RequestFunction(
             ?: request.queryParameters.getOrDefault(CLIENT_PARAMETER, "")
     }
 
+    class InvalidExternalPayloadException(val externalName: String) : RuntimeException()
```

Comment:
You should pass the `externalName` along to the `RuntimeException` constructor as the message so we see it when it gets thrown.

Reply:
woops meant to do that